hugegraph-api/src/main/java/com/baidu/hugegraph/config/ServerOptions.java

```diff
@@ -212,4 +212,27 @@ public static synchronized ServerOptions instance() {
                     disallowEmpty(),
                     "hugegraph:9fd95c9c-711b-415b-b85f-d4df46ba5c31"
             );
+    public static final ConfigOption<String> KEYSTORE_SERVER_FILE =
+            new ConfigOption<>(
+                    "server.keystore.file",
+                    "The server keystore file.",
```

Comment:
improve description: The path of server keystore file used when https protocol is enabled